packaging/rpm/cri-resource-manager.spec.in

```diff
@@ -28,14 +24,12 @@ make build BUILD_DIRS=cri-resmgr
 %make_install UNITDIR=%{_unitdir} SYSCONFDIR=%{_sysconfdir} BUILD_DIRS=cri-resmgr
 install -m 0700 -d %{?buildroot}%{_sharedstatedir}/cri-resmgr
 
-
 %files
 %defattr(-,root,root,-)
 %{_bindir}/*
 %{_sysconfdir}/sysconfig/*
 %{_unitdir}/*
 %dir %attr(0700,root,root) %{_sharedstatedir}/cri-resmgr
-
-%files doc
-%defattr(-,root,root,-)
-%doc CONTRIBUTING.md LICENSE README.md SECURITY.md TODO.md
+%dir %attr(0700,root,root) %{_sysconfdir}/cri-resmgr
+%config %{_sysconfdir}/cri-resmgr/*
```

Comment:
just from testing in env for upgrades, it might be better %config(noreplace)

Reply:
True, fixed.